language/move-lang/src/errors/mod.rs

```diff
@@ -117,14 +117,26 @@ fn convert_loc(files: &Files<String>, file_mapping: &FileMapping, loc: Loc) -> (
     (id, Span::new(begin_index, end_index))
 }
 
-fn render_error(files: &Files<String>, file_mapping: &FileMapping, mut error: Error) -> Diagnostic {
-    let mk_lbl = |err: (Loc, String)| -> Label {
+fn render_error(
+    files: &Files<String>,
+    file_mapping: &FileMapping,
+    mut error: Error,
+) -> Diagnostic<FileId> {
+    let mk_secondary_lbl = |err: (Loc, String)| -> Label<FileId> {
+        let (id, span) = convert_loc(files, file_mapping, err.0);
+        Label::<FileId>::secondary(id, span).with_message(err.1)
```

Comment:
Do you need this explicit type argument on the `Label`? I'm surprised it doesn't infer it

Reply:
was a by product of working through the type changes, being explicit here helped show what changes needed to happen to update the methods, still a rust n00b.

Reply:
You should be able to refer them, if you remove the annotations. Give it a shot :)